client/src/API/api.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+
+import token from '../configs/token';
+
+const weatherAPIToken = axios.create({
+  baseURL: 'http://localhost:8000/'
+});
+
+export const getToken = async () => {
+  const { data: token } = await weatherAPIToken.get('/token');
+  console.log(token);
+};
+
+const weatherAPI = axios.create({
+  baseURL: 'https://pfa.foreca.com/api/v1/',
+  headers: { authorization: `Bearer ${token}` }
+});
+
+const lon = '23.9036';
+const lat = '54.8985';
```

Comment:
no need to store fixed position

Reply:
please make it work without hardcoded values, it's much easier to do it now then later with more code added

Reply:
Fixed hard coded values, now coordinates are given from getLocation() function